### DISCUSSION TITLE
TM1 Integrated Login with FLASK

### DISCUSSION THREAD

**Discussion by il66ezov** [cubewise-code/tm1py#539] [Ideas]
I want to make a Flask application where a frontend user clicks a button and the click executes the TM1py Integrated Login with the Windows data of the front end user. 
Is this possible?

**Version**
- TM1py 1.6.2
- TM1 Version 11.8.00300.3.4

Thank you for your help

**Comment by MariusWirtz** [comment]
Hi @il66ezov,

to be honest I don't know how to delegate the integrated login from within the browser towards the python / TM1py backend.
I'm curious if you find a good solution for this challenge.

To build a web app for TM1, you may want to look at Apliqo UX. It's a commercial product that has addressed all those challenges and gives you a toolbox to build a visual web interface for TM1.
https://apliqo.com/de/apliqoux/

**Comment by MariusWirtz** [comment]
I will move this topic into discussions, it's more a question than an issue IMO.
